SECURITY.md

```diff
@@ -1,4 +1,4 @@
 # Security Policy
 For a more in-depth look at our security policy, please check out our [Coordinated Vulnerability Disclosure Policy](https://openai.com/security/disclosure/#:~:text=Disclosure%20Policy,-Security%20is%20essential&text=OpenAI%27s%20coordinated%20vulnerability%20disclosure%20policy,expect%20from%20us%20in%20return.).
 
-Our PGP key can located [at this address.](https://cdn.openai.com/security.txt)
```

Comment:
Attention! This section of the SECURITY.md file is also touched by https://github.com/openai/evals/pull/12!

Reply:
Whoops, looks like I glossed over this. Why don't I stick this into my PR as I'm already changing it (and it seems a little more relevant).
@zfb132